testbed/testbed/validator.go

```diff
@@ -36,12 +36,43 @@ type TestCaseValidator interface {
 }
 
 // PerfTestValidator implements TestCaseValidator for test suites using PerformanceResults for summarizing results.
-type PerfTestValidator struct{}
+type PerfTestValidator struct {
+
+	// IsScraping specifies if the load generator
+	IsScraping bool
+
+	// ScrapeInterval is the interval the receiver is scraping at, if IsScraping is enabled
+	ScrapeInterval time.Duration
+
+	// ExtraItemsScrapedPerIntervalCount accounts for if the receiver adds additional metrics than those sent
```

Comment:
Unclear. I wonder if this is what is meant:
```suggestion
	// ExtraItemsScrapedPerIntervalCount accounts for if the receiver in the Collector adds additional metrics than those sent by the generator used by the testbed.
```